### DISCUSSION TITLE
[Question] Where is the (Python) API documentation?

### DISCUSSION THREAD

**Discussion by Mausy5043** [DanWBR/dwsim#634] [General]
I'm using DWSIM Pro and I'm trying to understand how to use the in-built IronPython to interface with my model.
In example code I've come across`DWSIM.GetProp()` like this one:
https://github.com/DanWBR/dwsim/blob/4ff9681d722f3e94a340e6219270c4082d6098e3/PlatformFiles/Common/FOSSEE/Stoichiometric_Reactor/Stoichometric_Reactor.txt#L19

So, I wonder what other properties I could extract from the model and I went searching for the API documentation. I found [this](https://dwsim.org/api_help/html/M_DWSIM_Thermodynamics_Streams_MaterialStream_GetProp_1.htm), but this is the documentation for the C# and VB API and doesn't provide much details. There is a button there that links to the source, but that link doesn't seem to be up-to-date; it points to a seemingly random part of the sources.
Any plans to update the documentation and maybe add a python API documentation?
Thanks!

**Comment by DanWBR** [comment]
@Mausy5043 please check this https://sourceforge.net/p/dwsim/discussion/scripting/thread/3f56e2f421/
